packages/ui-extensions-react/src/surfaces/point-of-sale.ts

```diff
@@ -0,0 +1,5 @@
+export {extend, extension} from '@shopify/ui-extensions/point-of-sale';
```

Comment:
Looking at [checkout example](https://github.com/Shopify/ui-extensions/blob/18efa584c8a3bd6f1b56620152fe74aa6ea99ac5/packages/ui-extensions-react/src/surfaces/checkout.ts#L65) I think we should also export all individual types from here.